src/main/java/io/vlingo/actors/InboundGridActorControl.java

```diff
@@ -114,4 +114,12 @@ public void relocate(Id receiver, Id sender, Definition.SerializationProxy defin
     });
     actor.resumeFromRelocation();
   }
+
+  @Override
+  public void recover(Id recipient, Id sender, Definition.SerializationProxy definitionProxy, Address address) {
+    logger.debug("Processing: Received application message: Recover");
+    final Optional<GridActor<?>> maybeActor = Optional.ofNullable((GridActor<?>) grid.actorAt(address));
+    final GridActor<?> actor = maybeActor.orElseGet(() -> (GridActor<?>) grid.actorAt(address));
```

Comment:
obviously there is a bug here :)

It should create an actor instance using `grid#actorFor`